build-tools/packages/build-cli/prettier.config.cjs

```diff
@@ -5,6 +5,7 @@
 
 module.exports = {
     ...require("@fluidframework/build-common/prettier.config.cjs"),
+    useTabs: false,
```

Comment:
Can we add the TODO comment here about removing when we come back to enable prettier globally?

Reply:
(Same with the other configs)

Reply:
I will add the comments next to the changes that I made. I was first trying to figure out what the problem was with the `Pull Request Labeler` error and then add the comments.

I guess I could've just included the comments at the same time :D. Thanks Josh!